app/src/main/graphql/com/cornellappdev/android/eatery/ctEateryQuery.graphql

```diff
@@ -0,0 +1,34 @@
+query AllCtEateries {
+    collegetownEateries {
+      	address        
+        categories
+        coordinates {
+            latitude
+            longitude
+        }
+        eateryType
+        id
+        imageUrl
+        name
+        paymentMethods {
+            brbs
+            cash
+            cornellCard
+            credit
+            mobile
+            swipes
+        }
+        phone
+        price
+        rating
+        url
+        operatingHours{
+          date
+          events{
+            startTime
+            endTime
+            description
+          }
+        }
+    }
+}
```

Comment:
Correct me if I'm wrong but doesn't the schema.json file need to be rebuilt in order for the GraphQL queries to work in code? If so it might be easiest if it's uploaded to git with this pr. https://www.apollographql.com/docs/ios/downloading-schema.html